WalkingApp/model.py

```diff
@@ -25,29 +25,37 @@ def get_recent_walks():
     Return the last 4 walks from the database as a list of tuples (date, kms)
     """
     with _open_database() as conn:
-        return [tuple(row) for row in conn.execute("SELECT date, kms FROM WALKS ORDER BY date DESC LIMIT 4")]
+        return [tuple(row) for row in conn.execute("SELECT day, kms FROM WALKS ORDER BY day DESC LIMIT 4")]
 
 
-def save_to_database(date, kms: float, time, kcal: int, steps: int):
+def save_to_database(day, kms: float, time, kcal: int, steps: int):
     """
     Insert a new walk into the database.
-    :param date: date value in the "%d/%m/%y" format
+    :param day: date value in the "%d/%m/%y" format
     :param kms: kilometres
     :param time: duration expressed as the number of minutes
     :param kcal: amount of kcal
     :param steps: amount of steps
     """
     with _open_database() as conn:
-        conn.execute("INSERT INTO WALKS VALUES (?, ?, ?, ?, ?)", [date, kms, time, kcal, steps])
+        conn.execute("INSERT INTO WALKS VALUES (?, ?, ?, ?, ?)", [day, kms, time, kcal, steps])
         conn.commit()
 
 
-def calculate_statistics():
+def calculate_overall_statistics():
     """
     Return the sum of kms, duration, kcal and steps from the database as a tuple.
     """
     with _open_database() as conn:
         return tuple(conn.execute("SELECT SUM(kms), SUM(duration), SUM(kcal), SUM(steps) FROM WALKS").fetchone())
 
 
-selected_date = None
+def calculate_monthly_statistics():
+    """
+    Return the monthly sum of kms, duration, kcal and steps from the database as a tuple.
+    """
+    with _open_database() as conn:
+        return tuple(conn.execute("""SELECT day, CURRENT_DATE, SUM(kms), SUM(duration), SUM(kcal), SUM(steps) FROM WALKS 
```

Comment:
You don't need to select `day` and `CURRENT_DATE`, just use them in the WHERE clause.

Reply:
Yeah, that's my mistake, I forgot to delete it - I put it there for my control, that the code returns me what I need for further work with the code